UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs

```diff
@@ -333,12 +333,14 @@ public void BodyPartRemoveHealthMaster()
 		/// </summary>
 		public void TryRemoveFromBody(bool beingGibbed = false)
 		{
+			bool alreadyBleeding = false;
 			SetRemovedColor();
 			foreach (var bodyPart in HealthMaster.BodyPartList)
 			{
-				if (bodyPart.BodyPartType == BodyPartType.Chest)
+				if (bodyPart.BodyPartType == BodyPartType.Chest && alreadyBleeding == false)
```

Comment:
isn't ``alreadyBleeding`` always ``false``?

Reply:
Aah, I understand it now.

Reply:
nope, for some reason BodyPartType.Chest is found five times, and adding this check causes it to only make bleeding happen once